ios-swift/CountlyTestApp-iOS-swift/AppDelegate.swift

```diff
@@ -18,14 +18,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate
         config.host = "https://YOUR_COUNTLY_SERVER"
         config.enableDebug = true
 
-//      config.features = [CLYPushNotifications, CLYCrashReporting, CLYAutoViewTracking] //Optional features
+//      config.features = [CLYFeature.pushNotifications, CLYFeature.CLYCrashReporting, CLYFeature.CLYAutoViewTracking] //Optional features
 
-//      config.launchOptions = launchOptions                            //Prior to v16.10 was required for CLYPushNotifications
-//      config.isTestDevice = true                                      //Optional marking as test device for CLYPushNotifications
-//      config.sendPushTokenAlways = ture                               //Optional forcing to send token always
+//      config.launchOptions = launchOptions                            //Prior to v16.10 was required for CLYFeature.pushNotifications
+//      config.isTestDevice = true                                      //Optional marking as test device for CLYFeature.pushNotifications
+//      config.sendPushTokenAlways = true                               //Optional forcing to send token always
 //      config.doNotShowAlertForNotifications = true                    //Optional disabling alerts shown by notification
 
-//      config.crashSegmentation = ["SomeOtherSDK":"v3.4.5"];           //Optional crash segmentation for CLYCrashReporting
+//      config.crashSegmentation = ["SomeOtherSDK":"v3.4.5"];           //Optional crash segmentation for CLYFeature.CLYCrashReporting
```

Comment:
For Crash reporting feature use this code "CLYFeature.crashReporting" instead of "CLYFeature.CLYCrashReporting"

Reply:
done